ansible/group_vars/all.yml

```diff
@@ -1,14 +1,15 @@
 ---
 ssh_key_name: ""
 target_dns_zone: ""
-target_subdomain: ""
+aws_custom_prefix: ""
```

Comment:
Unsure about this changing from target_subdomain to aws_custom_prefix, not sure they will mean the same in all contexts.

aws_custom_prefix I envision being used for a tag of ec2 resources
target_subdomain is used for the generated hostname and routes